08LessonEight/in-class-project/homework.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Homework</title>
+    <link href="css/homework-style.css" rel="stylesheet" type="text/css"/>
+  </head>
+
+  <body>
+    <div>
+      <div id="title">
+        <p class="b1">Recent From Blog</p>
+        <p>Donec tincidunt felis quam, eu tempus purus finibus in, Curabitur hendrerit, odio in viverra interdum, lorem velit scelerisque ipsum, a sagittis ligula leo in dolor. Etiam vestibulum.</p>
+      </div>
+
+      <div id="main">
+        <div class="article">
+          <img src="img/keyboard.JPG"/>
+          <p class="b2">Morbi eu metus ut libero laoreet vulputate</p>
+          <p>Proin a libero eleifend, posuere ipsum id, pulvinar ipsum. Praesent sit amet massa suscipit, ornare mi et, ultrices nisl. Vivamus venenatis</p>
+          <a href="#">Read More</a>
+        </div>
+        <div>
+          <div class="article">
+            <img src="img/laptop.jpg"/>
+            <div>
+              <p class="b2">Suspendisse a sagittis</p>
+              <p>Sed ut sem elit. Pellentesque vitae erat vehicula quam posuere porta</p>
+            </div>
+          </div>
+          <div class="article">
+            <img src="img/desk.jpg"/>
+            <div>
+              <p class="b2">Morbi eu metus ut libero</p>
+              <p>Pellentesque magna dolor, dapibus vitae lorem in , laoreet sagittis lacus.</p>
+            </div>
+          </div>
+          <div class="article">
+            <img src="img/window.jpg"/>
+            <div>
+              <p class="b2">Pellentesque magna dolor</p>
+              <p>Suspendisse a sagittis tortor, ac dapibus turpis.</p>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div id="footer">
+        <div>
+          <div class="footer-content">
+            <p class="b2">Why Profession?</p>
```

Comment:
everything looks great! more commenting please. Also, you'll stand at a 90% on this until you get those checkmarks in here and the right color. let me know if you need help with that.